apps/ledger-live-desktop/tests/specs/speculos/send.tx.spec.ts

```diff
@@ -1,48 +1,265 @@
 import { test } from "../../fixtures/common";
 import { Account } from "../../enum/Account";
+import { Fee } from "../../enum/Fee";
 import { Transaction } from "../../models/Transaction";
 import { specs } from "../../utils/speculos";
 import { addTmsLink } from "tests/utils/allureUtils";
 import { getDescription } from "../../utils/customJsonReporter";
 
-// ONLY TESTNET (SEND WILL BE APPROVED ON DEVICE)
-const transactions = [
-  //TODO: Reactivate when fees will be stable
-  //new Transaction(Account.tBTC_1, Account.tBTC_2, "0.00001", "medium"),
-  new Transaction(Account.sep_ETH_1, Account.sep_ETH_2.address, "0.00001", "medium"),
+const transaction = new Transaction(Account.sep_ETH_1, Account.sep_ETH_2, "0.00001", Fee.MEDIUM);
```

Comment:
can you please move inside the test const that have only one value to improve readability of the tests?